src/script/descriptor.cpp

```diff
@@ -243,6 +261,12 @@ class ConstPubkeyProvider final : public PubkeyProvider
         ret = EncodeSecret(key);
         return true;
     }
+    bool ToNormalizedString(const SigningProvider& arg, std::string& ret, bool priv) const override
```

Comment:
9be1437c49f986e8ed964d5f863b4bbcec340751 here and L218, maybe name the arg `out` like the header file declaration and the other definitions
```suggestion
    bool ToNormalizedString(const SigningProvider& arg, std::string& out, bool priv) const override
```